streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedKeyValueBufferTest.java

```diff
@@ -335,59 +342,64 @@ public void shouldRestoreOldFormat() {
 
         context.setRecordContext(new ProcessorRecordContext(0, 0, 0, "", null));
 
+        final Serializer<Change<String>> serializer = FullChangeSerde.castOrWrap(Serdes.String()).serializer();
+
+        final byte[] doomedValue = serializer.serialize(null, new Change<>("doomed", null));
+        final byte[] qwerValue = serializer.serialize(null, new Change<>("qwer", null));
+        final byte[] eo4imValue = serializer.serialize(null, new Change<>("eo4im", null));
```

Comment:
This is ok, since we have a prior knowledge that the actual value type has always been Change.

Reply:
I do need to add a test making sure we can convert from the v1 format, though.

Reply:
this is done now